tests/output/t523-operator-f.out

```diff
@@ -0,0 +1,54 @@
+Composite CeedOperator
+SubOperator [0]
+  3 Fields
+  2 Input Fields
+    [0] Operator Field "_weight"
+      Lmode "no transpose"
+      No vector
+    [1] Operator Field "dx"
+      Lmode "no transpose"
+      Active vector
+  1 Output Field
+    [0] Operator Field "_weight"
+      Lmode "no transpose"
+      Collocated basis
+SubOperator [1]
+  3 Fields
+  2 Input Fields
+    [0] Operator Field "_weight"
+      Lmode "no transpose"
+      No vector
+    [1] Operator Field "dx"
+      Lmode "no transpose"
+      Active vector
+  1 Output Field
+    [0] Operator Field "_weight"
+      Lmode "no transpose"
+      Collocated basis
```

Comment:
I welcome any feedback into the exact appearance of these outputs, but now we can view the basic data about any Ceed object.

Reply:
I like it!
I would personally only put colons to explain the following list of objects, like
```
SubOperator [0]:
  3 Fields
  2 Input Fields:
```
Good job!

Reply:
Cool, how about now?

Reply:
It helps with my OCD 😁 but it was really ok if you preferred the other way! Thanks. Nice job again.